stories/core/ColorPicker.stories.tsx

```diff
@@ -206,3 +206,88 @@ export const Advanced: Story<ColorPickerProps> = (args) => {
 };
 
 Advanced.args = {};
+
+export const Opacity: Story<ColorPickerProps> = (args) => {
```

Comment:
Maybe should be called `WithAlpha` or something

Reply:
Renamed